src/frontend/src/handler/mod.rs

```diff
@@ -1108,3 +1114,115 @@ pub async fn handle(
         _ => bail_not_implemented!("Unhandled statement: {}", stmt),
     }
 }
+
+fn check_ban_ddl_for_iceberg_engine_table(
```

Comment:
Did we ban sink into table for iceberg table as well?

Reply:
I think we shouldn't ban sink into table, because it should work based on our design.

Reply:
Then what about adding a test case for sink into table?